docs/assets/templates/notebooks/template-big.dockstore.yml

```diff
@@ -0,0 +1,74 @@
+# The first line refers to the version 1.2 of the .dockstore.yml schema
+version: 1.2
+
+# An array of notebooks. Each element corresponds to a notebook on Dockstore.
+notebooks:
+  
+  # The optional notebook name for a notebook, which may only consist of alphanumerics
+  # and internal underscores and hyphens, but no spaces or other characters. Names may not exceed 256 characters.
+  # If using a .dockstore.yml with multiple notebooks, this field is required 
+  # to uniquely identify notebooks in the repository.
+  #
+  # It should be noted that having the name come first is an arbitrary decision.
+  # You could use subclass instead, for instance. Provided arrays are not broken
+  # up, the order of fields within a .dockstore.yml is not important.
+  - name: <String>
+    
+    # The format used for the notebook. Currently, only JUPYTER is supported.
+    # This cannot be changed once the notebook is registered.
+    format: <JUPYTER>
+
+    # The programming language of the code contained in the notebook.
+    language: <PYTHON | R | JAVASCRIPT | SCALA | JULIA | OTHER>
+
+    # The absolute path to the notebook file in the Git repository.
+    # - For Jupyter notebooks, the notebook is a .ipynb file.
+    path: <String>
+
+    # The kernel image reference for the notebook.
+    kernel: <String>
```

Comment:
TLDR: For now, I'd suggest we not detail the `kernel` field in the 1.14 documentation, and instead tell anyone that needs it how to use it.

The rationale: At the moment, the `kernel` field exists to support the LBL use case.  I've been pondering it's wider use, and am instead leaning towards us preferring the REES-based software dependencies.  If people register a bunch of notebooks that depend on particular kernel images, they'll be tied to the particular platforms (not many) that support them.   I have a different idea, will present at a story time in a week or three...